packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,362 @@
+              <CategoryWithDapps category={category} onDappPress={onDappPress} />
+            ))}
+        </>
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+function CategoryWithDapps({ category: categoryWithApps, onDappPress }: CategoryWithDappsProps) {
+  Logger.debug(TAG, `Render category ${JSON.stringify(categoryWithApps)}`)
+
+  return (
+    <View style={styles.categoryContainer} key={`category-${categoryWithApps.id}`}>
+      <View
+        style={{
+          ...styles.categoryTextContainer,
+          backgroundColor: categoryWithApps.backgroundColor,
+        }}
+      >
+        <Text style={{ ...styles.categoryText, color: categoryWithApps.fontColor }}>
+          {categoryWithApps.name}
+        </Text>
+      </View>
+      <>
+        {categoryWithApps.dapps.map((dapp) => (
+          <Dapp dapp={dapp} onDappPress={onDappPress} />
+        ))}
+      </>
+    </View>
+  )
+}
+
+function Dapp({ dapp, onDappPress }: DappProps) {
+  Logger.debug(TAG, `Render item ${JSON.stringify(dapp)}`)
+
+  const onPress = () => {
+    return onDappPress(dapp)
+  }
+
+  return (
+    <Card style={styles.card} rounded={true} shadow={Shadow.Soft} key={`item-${dapp.id}`}>
+      <TouchableOpacity style={styles.pressableCard} onPress={onPress}>
+        <Image source={{ uri: dapp.iconUrl }} style={styles.dappIcon} />
+        <View style={styles.itemTextContainer}>
+          <Text style={styles.itemTitleText}>{dapp.name}</Text>
+          <Text style={styles.itemSubtitleText}>{dapp.description}</Text>
+        </View>
+        <LinkArrow style={styles.linkArrow} />
+      </TouchableOpacity>
+    </Card>
+  )
+}
+
+const styles = StyleSheet.create({
+  safeAreaContainer: {
+    flex: 1,
+  },
+  scrollContainer: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  centerContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    flex: 1,
+  },
+  categoryContainer: {
+    alignItems: 'flex-start',
+    justifyContent: 'center',
+    flex: 1,
+    marginVertical: 16,
+    paddingHorizontal: variables.contentPadding,
+  },
+  itemTextContainer: {
+    flex: 1,
+  },
+  descriptionContainer: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    flex: 1,
+    paddingHorizontal: variables.contentPadding,
+    marginHorizontal: 8,
+    marginTop: 16,
+  },
+  helpIconContainer: {
+    padding: variables.contentPadding,
+  },
+  loadingIcon: {
+    marginVertical: 20,
+    height: 108,
+    width: 108,
+  },
+  dappIcon: {
+    width: 40,
+    height: 40,
+    borderRadius: 8,
+    marginRight: 16,
+  },
+  pressableCard: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    flex: 1,
+  },
+  card: {
+    marginTop: 16,
+    flex: 1,
+    alignItems: 'center',
+  },
+  categoryText: {
+    ...fontStyles.regular,
+    fontSize: 12.5,
+    borderRadius: 100,
+    paddingHorizontal: 11,
+    paddingVertical: 4,
+  },
+  itemTitleText: {
+    ...fontStyles.small,
+    fontWeight: '500',
```

Comment:
I forgot the exact details now, but there are differences between iOS and Android related to `fontWeight` and our custom font, so this might not render the way we expect.
We should stick to the styles defined in our design system to avoid this.

https://github.com/valora-inc/wallet/blob/66cdc5a6bd34cd0957e968d599e223cccaf6496e/packages/react-components/styles/fonts.tsx#L38-L115

Reply:
I added that to honor the design parameters. But I just removed the `fontWeight` property and it's showing the same, so we can remove it.